docs/articles/toc.yml

```diff
@@ -1,66 +1,76 @@
 - name: Introduction
   href: intro.md
-- name: Getting started
+- name: The Basics
   items:
-  - name: Bot application
-    href: getting_started/bot_application.md
-  - name: Basic bot
-    href: getting_started/basic_bot.md
-  - name: Alternate WebSocket clients
-    href: getting_started/alternate_ws.md
-  - name: Event reference
-    href: getting_started/event_reference.md
-  - name: Notes about using Mono
-    href: getting_started/mono.md
+  - name: Creating a Bot Application
+    href: basics/bot_application.md
+  - name: Writing a Basic Bot
+    href: basics/basic_bot.md
+- name: Beyond Basics
+  items:
+  - name: Understanding TAP Events
+    href: beyond_basics/event_reference.md
+  - name: Alternate WebSocket Clients
+    href: beyond_basics/alternate_ws.md
+  - name: Using the Mono Runtime
```

Comment:
Axe it, maybe?

Reply:
just as confirmation, axe both the basic_bot.md and bot_application.md

Reply:
just wanted to make sure i understood correctly

Reply:
I was referring to the Mono section specifically. We should not encourage using Mono.

Reply:
I absolutely agree.  We will remove it tonight when we do the sweep of removing things that are no longer supported such as win7, etc

Reply:
SO I didnt axe it completely.  I left the page there but i rewrote the page to emphasize that they not use it and I removed all code that was there or guide.  If you are okay with how i wrote it then ill leave it be, if you still want it axed, then we can remove it

Reply:
I feel we should just nix the article; no point in having one for something we don't officially support.

Reply:
I would typically agree but with the amount of people that ask/bang their heads about it, i think that we should atleast mention it.  If we want to get rid of it, i say that we on the intro page, state what we do support and what we dont.

Reply:
Yeah, something on either the index or the intro page would be a smart idea.

Reply:
If it's a mention akin to "not supported, therefore unrecommended; you will not receive help", maybe with reasoning, it's ok to keep it.

Reply:
I believe that adding the following to the intro will suffice with dropping the mono page.

## Supported/Unsupported Runtimes/Enviorments
With regards of supported frameworks and specific version, we support anything the the .Net team at Microsoft supports.  At the time of writing, we support .Net 4.6.2 and above along with .Net Core 2.2 and above along with any OS that those support.  Rule of thumb should always be, if Microsoft supports it fully, then we support it as well.  

Other frameworks such as Mono or running on hardware such as a Raspberry Pi 0/1 will not be supported due to legacy or flaws associated with them.  Although you are more than welcome to ~~bang your head in circles~~ attempt to use these hardware/frameworks, you will not under any circumstance get any support when things break.

Reply:
> we support .Net 4.6.2

Note that we do not explicitly support it. As a rule of thumb, we provide explicit support for latest non-preview and LTS versions of .NET Core (at the time of writing, Core 3.1 and 2.1). We do not provide support for .NETFX. Other targets (Xamarin, et al) are hit and miss. They might work, but we don't do any testing.

Reply:
That is a very good point.  I totally reworked the opening paragraph then:

# Supported/Unsupported Runtimes/Environments
With regards of supported frameworks and specific versions, we currently target .Net Standard 2.0 (.Net Standard 1.3 is being dropped shortly).  With this said we formally support the latest non-preview and LTS versions of .Net Core, however any other targets such as .NETFX, Xamarin, etc may work perfectly but it is untested.  

Other frameworks such as Mono or running on hardware such as a Raspberry Pi 0/1 will not be supported due to legacy or flaws associated with them. Although you are more than welcome to ~~bang your head in circles~~ attempt to use these hardware/frameworks, you will not under any circumstance get any support when things break.